src/models/deprecated.ts

```diff
@@ -0,0 +1,14 @@
+import { SearchError } from './answersapi/SearchError';
+import { SearchRequest } from './core/SearchRequest';
+
+/**
+ * @deprecated AnswersError is deprecated and has been replaced by SearchError
+ * @public
+ */
+export type AnswersError = SearchError;
+
+/**
+* @deprecated AnswersRequest is deprecated and has been replaced by SearchRequest
+* @public
+*/
+export type AnswersRequest = SearchRequest;
```

Comment:
using types for the synonyms could be a potentially breaking change. since `AnswersRequest` used to be an interface, defining it as a type here might break the implementation of anyone who previously used to extend `AnswersRequest`, because types can't be extended. I think it would be better to try to have the synonyms match if possible (i.e. interfaces have interface synonyms, classes have class synonyms, etc)

Reply:
Done!